### ISSUE TITLE
Trashbin bug and suggestions

### ISSUE THREAD

**Issue by luke7263** [sjarvela/kloudspeaker#27]
I noticed a bug, coming when viewing thumbnail in Thashbin folder. I think it is important to be able to view thumbnails while in trashbin. But admin user can view thumbnails ! Only some rights pb somewhere ..

KLOUDSPEAKER ERROR: ServiceException: INVALID_REQUEST=Trash access not allowed

Still another remark for trashbin: since the trashbin is now implemented, the deletion confirmation message should be canceled (with an tuning option).

And about the quota for trashbin, a simple thing could be adding an option for users like "keep_files_in_trashbin = number-of-days", and you would clean the trashbin at each user connection ? So no need for cron or admin action ..

**Comment by sjarvela** [comment]
Yes, I'll check out the thumbnail access issue, I know what it is about, just have to think about how to overcome it (there is a reason for the access restriction).

Good suggestions, I'll add in in future release.

**Comment by sjarvela** [comment]
Thumbnail and access improvements in next release.

**Comment by sjarvela** [comment]
Also first draft of trash expiration in next release
